Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -1019,14 +1017,11 @@
       - ClothingUniformJumpskirtScientist
       - ClothingUniformJumpsuitSeniorResearcher
       - ClothingUniformJumpskirtSeniorResearcher
-      - ClothingHeadHatBeretSecurity
       - ClothingUniformJumpsuitSec
       - ClothingUniformJumpskirtSec
       - ClothingHeadHatBeretBrigmedic
```

Comment:
You forgot this one

Reply:
due to irl circumstances i cannot change any code on this PR whatsomever, if a maintainer wishes to make any changes, they can do so